src/main/java/org/cactoos/map/Merged.java

```diff
@@ -42,12 +43,22 @@
      */
     @SafeVarargs
     public Merged(final Map<? extends K, ? extends V>... maps) {
+        this(new IterableOf<>(maps));
+    }
+
+    /**
+     * Ctor.
+     * @param maps Maps to merge.
```

Comment:
@andreoss I suggest to write as description : `Iterable of {@link Map}s`

Reply:
@baudoliver7 Fixed.